interactions/client/bot.py

```diff
@@ -1340,7 +1340,7 @@ def decorator(func: Coroutine):
 
     if func:
         # allows omitting `()` on `@listener`
-        func.__listener_name__ = func.__name__
+        func.__listener_name__ = func.__name__ if not name else name
```

Comment:
```py
func.__listener_name__ = name or func.__name__
```

Reply:
Suggestion: make it a suggestion so I can just commit it

Reply:
How do you do that?

Reply:
On deskotp its a file symbol with ± inside, when you select one or multiple lines, on mobile its just ±

Reply:
Done